algorithms/simon_algorithm.ipynb

```diff
@@ -432,10 +428,9 @@
      "text": [
       "The hidden bistring s[ 0 ], s[ 1 ]....s[ 5 ] is the one satisfying the following system of linear equations:\n",
```

Comment:
I think it would be nice (if you want) to have a final message printed saying explicitly what's the result, as in https://learning.quantum.ibm.com/course/fundamentals-of-quantum-algorithms/quantum-query-algorithms#implementation-3.

Reply:
I added concluding statements about what can be inferred from the experiment results and how it is significantly faster than classical computers.  Please let me know if there is anything else.